src/vivarium_gates_shigella_vaccine/data/extract.py

```diff
@@ -39,6 +39,18 @@ def load_forecast_from_xarray(path: Path, location_id: int) -> pd.DataFrame:
             .reset_index())
 
 
+def get_location_specific_life_expectancy(location_id: int):
+    """Loads formatted country specific life expectancy table."""
+
+    path = Path(__file__).resolve().parent / "life_expectancy_with_forecasted_data_12.23.19.csv"
+    df = pd.read_csv(path, index_col=False)
+    df = df.drop(['Unnamed: 0'], axis=1)  # Old index, why can't I avoid reading it in?
+    df = df.rename(columns={'ex_inc': 'value'})
```

Comment:
when I wrote the life expectancy transforms, I matched the data already in the artifact. That data didn't have a value column, it had a life_expectancy column. Maybe this was addressed in subsequent lib versions